src/pyinstaller_versionfile/__init__.py

```diff
@@ -36,19 +37,29 @@ def create_versionfile(
     __create(metadata, output_file)
 
 
-def create_versionfile_from_input_file(output_file, input_file, version=None):
+def create_versionfile_from_input_file(
+    output_file,
+    input_file,
+    version=None,
+    **kwargs,
```

Comment:
Are you sure about **kwargs? Why not just continue the list with separate parameters? The list is manageably long. IDE users would certainly welcome this.

Reply:
Good hint. Changed it in the last diff. At least the "public" API (i.e. `__init__.py`) now has explicitly specified parameters.